MAINTAINERS.md

```diff
@@ -1,17 +0,0 @@
-# Maintainers
```

Comment:
@jezdez is this correct to move the governance stuff from the top of the repo into a subdirectory of the docs directory?

Reply:
No, you're right, reverting.